adapters/repos/db/helpers/tokenizer.go

```diff
@@ -150,18 +170,67 @@ func tokenizeGSE(in string) []string {
 	defer gseTokenizerLock.Unlock()
 	terms := gseTokenizer.CutAll(in)
 
-	// Remove empty strings from terms
-	for i := 0; i < len(terms); i++ {
-		if terms[i] == "" || terms[i] == " " {
-			terms = append(terms[:i], terms[i+1:]...)
-			i--
-		}
-	}
+	terms = removeEmptyStrings(terms)
 
 	alpha := tokenizeWord(in)
 	return append(terms, alpha...)
 }
 
+type KagomeTokenizers struct {
+	Korean *kagomeTokenizer.Tokenizer
+}
+
+var (
+	tokenizers     KagomeTokenizers
+	kagomeInitLock sync.Mutex
+)
+
+func InitializeKagomeTokenizerKr() error {
+	// Acquire lock to prevent initialization race
+	kagomeInitLock.Lock()
+	defer kagomeInitLock.Unlock()
+
+	if os.Getenv("ENABLE_KOREAN_TOKENIZER") == "true" {
```

Comment:
Can we just enable this tokenizer by default?

Reply:
oh I can see that we do this check also from GSE tokenizer, hmm, maybe let's keep this